format/Schema.fbs

```diff
@@ -267,7 +267,9 @@ table KeyValue {
 
 /// ----------------------------------------------------------------------
 /// Dictionary encoding metadata
-
+/// Maintained for forwards compatibility, in the future
+/// Dictionaries might be (sparse) maps betwen Index and Value.
```

Comment:
What is a "sparse map" here?

Reply:
Updated to "Kind".  I reworded as discontinuous indices.